__tests__/components/editor/PropertyComponent.test.js

```diff
@@ -98,11 +99,32 @@ describe('<PropertyComponent />', () => {
         }
       }
 
-      const wrapper = shallow(<PropertyComponent propertyTemplate={template} />)
+      const wrapper = shallow(<PropertyComponent propertyTemplate={template}
+                               reduxPath={["http://id.loc.gov/ontologies/bibframe/note"]}/>)
       expect(wrapper.find('Connect(InputListLOC)').length).toEqual(0)
       expect(wrapper.find('Connect(InputLookupQA)').length).toEqual(0)
       expect(wrapper.find('Connect(InputLiteral)').length).toEqual(0)
     })
   })
 
+  it('error if <PropertyComponent /> is missing reduxPath props', () => {
```

Comment:
I think this test description is missing the initial verb, like:

```javascript
it('logs an error if <PropertyComponent /> is missing the reduxPath property', () => {
```

Reply:
Fixed in commit.